src/Detector/Php/PhpPlatformDetectorResult.cs

```diff
@@ -0,0 +1,20 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Oryx.Detector.Php
+{
+    public class PhpPlatformDetectorResult : PlatformDetectorResult
+    {
+        public bool ComposerFileExists { get; set; }
+
+        public bool ComposerLockFileExists { get; set; }
+
+        public IDictionary<string, string> Dependencies { get; set; }
```

Comment:
It's a good idea to add this dependencies so that users can get information through it, seems that it didn't have implementation, maybe we could add its implementation in the future PR

Reply:
Good catch. I actually should have reverted this because initially I wanted to add 'Dependencies' here but later I realized that if we should include very 'build' specific stuff as part of detection logic itself. I know this is not an easy answer since I am sure there would be arguments for and against it. That said we need to work on designing these sub-types' structure also well since they are going to be part of the contract. For now, I will revert this change since it is not really being used.

Reply:
I see, let's revert it and take some time to think about it and design it in the future.